Wikipedia/Code/SearchViewController.swift

```diff
@@ -452,7 +512,7 @@ class SearchViewController: ThemeableViewController, WMFNavigationBarConfiguring
             guard let self else {
```

Comment:
Not certain the bug is in this area but I notice that tapping the search result opens in the existing tab rather than a new one. See my last navigation to "Cat":


https://github.com/user-attachments/assets/73687b67-31bc-41f1-a39d-fbe457a94a33